pack/tde/tde.json

```diff
@@ -298,42 +298,46 @@
 		"quantity": 1,
 		"restrictions": "investigator:06004",
 		"subtype_code": "weakness",
-		"text": "<b>Revelation</b> - If Dream-Gate <i>(Wonderous Journey)</i> is already in play, flip it over. Otherwise, search your bonded cards from Dream-Gate <i>(Pointless Reality)</i> and put it into play. In either case, disengage from each enemy engaged with you and move to Dream-Gate.",
+		"text": "<b>Revelation</b> - If Dream-Gate <i>(Wondrous Journey)</i> is already in play, flip it over. Otherwise, search your bonded cards from Dream-Gate <i>(Pointless Reality)</i> and put it into play. In either case, disengage from each enemy engaged with you and move to Dream-Gate.",
 		"traits": "Madness.",
 		"type_code": "treachery"
 	},
 	{
 		"back_link": "06015b",
+		"bonded_to": "06013",
 		"clues": 0,
 		"code": "06015a",
-		"double_sided": true,
 		"faction_code": "neutral",
 		"illustrator": "Frej Agelii",
 		"name": "Dream-Gate",
 		"pack_code": "tde",
 		"position": 15,
 		"quantity": 1,
+		"restrictions": "investigator:06004",
```

Comment:
Given this card is bonded (and also a location) I'm not sure that the restriction is necessary?  It's not actually in his deck, just on the side?

Reply:
I've removed it - thank you!